source/Sashimi.Aws.Accounts/PropertyDictionaryExtensions.cs

```diff
@@ -1,15 +1,18 @@
 using System.Collections.Generic;
 
-static class PropertyDictionaryExtensions
+namespace Sashimi.Aws.Accounts
 {
-    public static bool ContainsPropertyWithValue(this IDictionary<string, string> dictionary, string key)
+    static class PropertyDictionaryExtensions
     {
-        if (!dictionary.ContainsKey(key))
+        public static bool ContainsPropertyWithValue(this IDictionary<string, string> dictionary, string key)
```

Comment:
Consider renaming this global extension to:
```suggestion
        public static bool ContainsPropertyWithNonEmptyValue(this IDictionary<string, string> dictionary, string key)
```